config/settings/local.py

```diff
@@ -23,44 +23,6 @@
     }
 }
 
-# EMAIL
-# ------------------------------------------------------------------------------
-# https://docs.djangoproject.com/en/dev/ref/settings/#email-host
-EMAIL_HOST = env("EMAIL_HOST", default="mailhog")
```

Comment:
no need to remove these, locally we will need the `debug_toolbar` etc

Reply:
we need an email locally? why?